<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paciente_tarjeta_cita</class>
 <widget class="QWidget" name="paciente_tarjeta_cita">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>9999</width>
     <height>9999</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f4f4f4;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>69</y>
     <width>98</width>
     <height>99</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/imgCita.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fecha_cita">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>22</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <property name="text">
    <string>Fecha Cita</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nombre_medico">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>51</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <property name="text">
    <string>Nombre Doctor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="info_cita">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>110</y>
     <width>280</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
background: none;
border: 1px solid #C4C4C4;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Informacion Cita Medica</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_horario">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <property name="text">
    <string>Horario</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gestionar">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>188</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: white;
	background-color: #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;

}</string>
   </property>
   <property name="text">
    <string>Gestionar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
